<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../writernote.qrc">
    <normaloff>:/image/snap/gui/icon-writernote.png</normaloff>:/image/snap/gui/icon-writernote.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidgetSX">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentRow">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_left_hide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../writernote.qrc">
            <normaloff>:/image/images/icon-hide-left.png.png</normaloff>:/image/images/icon-hide-left.png.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_right_hide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../writernote.qrc">
            <normaloff>:/image/images/icon-hide-right.png</normaloff>:/image/images/icon-hide-right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="layouteditor">
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="startriascoltobotton">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../writernote.qrc">
                <normaloff>:/image/images/play.png</normaloff>:/image/images/play.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopriascolto">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../writernote.qrc">
                <normaloff>:/image/images/stopRecord_.png</normaloff>:/image/images/stopRecord_.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="audioSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="volumeSlider">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2_stile">
      <item>
       <widget class="QSpinBox" name="spinBox_fontsize">
        <property name="minimum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontComboBox_fonttipo"/>
      </item>
      <item>
       <widget class="QToolButton" name="boldbotton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../writernote.qrc">
          <normaloff>:/image/images/bold.png</normaloff>:/image/images/bold.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="inserttablebotton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../writernote.qrc">
          <normaloff>:/image/images/table.png</normaloff>:/image/images/table.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="listbotton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../writernote.qrc">
          <normaloff>:/image/images/bullet.png</normaloff>:/image/images/bullet.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="insertimagebotton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../writernote.qrc">
          <normaloff>:/image/images/image.png</normaloff>:/image/images/image.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWriternote">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPrint"/>
    <addaction name="actiontopdf"/>
    <addaction name="startrecording"/>
    <addaction name="stoprecordingbotton"/>
    <addaction name="actionAuto_Save"/>
    <addaction name="actionEnable_redo_undo"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndu"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuAudio_option">
    <property name="title">
     <string>Audio option</string>
    </property>
    <addaction name="actionListen_current_audio"/>
    <addaction name="actionCreate_new_copybook"/>
    <addaction name="actionDelete_audio"/>
    <addaction name="actionDelete_copybook"/>
    <addaction name="actionConvert_audio_to_text"/>
    <addaction name="actionQuality_setting"/>
    <addaction name="actionDefault_save_location"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSource_code"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionCheck_update"/>
    <addaction name="actionBug_report"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionnew_sheet"/>
   </widget>
   <addaction name="menuWriternote"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAudio_option"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionSave_As"/>
   <addaction name="actioncloud"/>
   <addaction name="actionPrint"/>
   <addaction name="startrecording"/>
   <addaction name="stoprecordingbotton"/>
   <addaction name="pauserecordingbotton"/>
   <addaction name="actionImport_Video"/>
   <addaction name="actioncompress_video"/>
  </widget>
  <widget class="QToolBar" name="simpleactionbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndu"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QToolBar" name="audiobar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionListen_current_audio"/>
   <addaction name="actionDelete_audio"/>
   <addaction name="actionDelete_copybook"/>
   <addaction name="actionCreate_new_copybook"/>
   <addaction name="actionConvert_audio_to_text"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolbarmatita">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionpen"/>
   <addaction name="actionrubber"/>
   <addaction name="actionselezionetext"/>
   <addaction name="actioninsertText"/>
  </widget>
  <widget class="QToolBar" name="toolBarcolore">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionblack"/>
   <addaction name="actionwhite"/>
   <addaction name="actionyellow"/>
   <addaction name="actionred"/>
   <addaction name="actionchoose_color"/>
   <addaction name="actioninsertImage"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolSheet">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew_sheet"/>
  </widget>
  <action name="actionNew_File">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme=":/image/images/newFileBig.png" resource="../writernote.qrc">
     <normaloff>:/image/images/newFileBig.png</normaloff>:/image/images/newFileBig.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/disk.png</normaloff>:/image/images/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/disk--pencil.png</normaloff>:/image/images/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/printer.png</normaloff>:/image/images/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="startrecording">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/recoding.png</normaloff>:/image/images/recoding.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionImport_Video">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/importVideo.png</normaloff>:/image/images/importVideo.png</iconset>
   </property>
   <property name="text">
    <string>Import Video</string>
   </property>
  </action>
  <action name="stoprecordingbotton">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/stopRecord_.png</normaloff>:/image/images/stopRecord_.png</iconset>
   </property>
   <property name="text">
    <string>Stop Record</string>
   </property>
  </action>
  <action name="actionImport_Video_2">
   <property name="text">
    <string>Import Video</string>
   </property>
  </action>
  <action name="actionUndu">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/arrow-curve-180-left.png</normaloff>:/image/images/arrow-curve-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/arrow-curve.png</normaloff>:/image/images/arrow-curve.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/document-copy.png</normaloff>:/image/images/document-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/scissors.png</normaloff>:/image/images/scissors.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset>
     <normaloff>../images/selection-input.png</normaloff>../images/selection-input.png</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/clipboard-paste-document-text.png</normaloff>:/image/images/clipboard-paste-document-text.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSource_code">
   <property name="text">
    <string>Source code</string>
   </property>
  </action>
  <action name="actionBug_report">
   <property name="text">
    <string>Bug report</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionListen_current_audio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/manoIcon.png</normaloff>:/image/images/manoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Listen current audio</string>
   </property>
  </action>
  <action name="actionDelete_copybook">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/deleteCopyBook.png</normaloff>:/image/images/deleteCopyBook.png</iconset>
   </property>
   <property name="text">
    <string>Delete copybook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCreate_new_copybook">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/newFile.png</normaloff>:/image/images/newFile.png</iconset>
   </property>
   <property name="text">
    <string>Create new copybook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionConvert_audio_to_text">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/text-speech.png</normaloff>:/image/images/text-speech.png</iconset>
   </property>
   <property name="text">
    <string>Convert audio to text</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/blue-folder-open-document.png</normaloff>:/image/images/blue-folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDelete_audio">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/deleteAudio.png</normaloff>:/image/images/deleteAudio.png</iconset>
   </property>
   <property name="text">
    <string>Delete audio</string>
   </property>
  </action>
  <action name="pauserecordingbotton">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/pause-recording.png</normaloff>:/image/images/pause-recording.png</iconset>
   </property>
   <property name="text">
    <string>pauserecording</string>
   </property>
  </action>
  <action name="actionQuality_setting">
   <property name="text">
    <string>Quality setting</string>
   </property>
  </action>
  <action name="actioncompress_video">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/compress-video.png</normaloff>:/image/images/compress-video.png</iconset>
   </property>
   <property name="text">
    <string>compress video</string>
   </property>
  </action>
  <action name="actionpen">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/pen-icon.png</normaloff>:/image/images/pen-icon.png</iconset>
   </property>
   <property name="text">
    <string>pen</string>
   </property>
  </action>
  <action name="actionrubber">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/rubber.png</normaloff>:/image/images/rubber.png</iconset>
   </property>
   <property name="text">
    <string>rubber</string>
   </property>
  </action>
  <action name="actionyellow">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/yellow-icon.png</normaloff>:/image/images/yellow-icon.png</iconset>
   </property>
   <property name="text">
    <string>yellow</string>
   </property>
  </action>
  <action name="actionwhite">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/white-icon.png</normaloff>:/image/images/white-icon.png</iconset>
   </property>
   <property name="text">
    <string>white</string>
   </property>
  </action>
  <action name="actionblack">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/black-icon.png</normaloff>:/image/images/black-icon.png</iconset>
   </property>
   <property name="text">
    <string>black</string>
   </property>
  </action>
  <action name="actionred">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/red-icon.png</normaloff>:/image/images/red-icon.png</iconset>
   </property>
   <property name="text">
    <string>red</string>
   </property>
  </action>
  <action name="actionselezionetext">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/cisors-cut.png</normaloff>:/image/images/cisors-cut.png</iconset>
   </property>
   <property name="text">
    <string>selezionedisegno</string>
   </property>
  </action>
  <action name="actiontopdf">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/pdf-icon.png</normaloff>:/image/images/pdf-icon.png</iconset>
   </property>
   <property name="text">
    <string>toppf</string>
   </property>
  </action>
  <action name="actionCheck_update">
   <property name="text">
    <string>Check update</string>
   </property>
  </action>
  <action name="actioninsertImage">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/image.png</normaloff>:/image/images/image.png</iconset>
   </property>
   <property name="text">
    <string>insertImage</string>
   </property>
  </action>
  <action name="actionnew_sheet">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/new-sheetpng</normaloff>:/image/images/new-sheetpng</iconset>
   </property>
   <property name="text">
    <string>Manage sheet</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Save</string>
   </property>
  </action>
  <action name="actionEnable_redo_undo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disable redo undo</string>
   </property>
  </action>
  <action name="actionchoose_color">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/choose-color-icon.png</normaloff>:/image/images/choose-color-icon.png</iconset>
   </property>
   <property name="text">
    <string>choose-color</string>
   </property>
  </action>
  <action name="actioncloud">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/cloud-icon.png</normaloff>:/image/images/cloud-icon.png</iconset>
   </property>
   <property name="text">
    <string>cloud</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actioninsertText">
   <property name="icon">
    <iconset resource="../writernote.qrc">
     <normaloff>:/image/images/insertText.png</normaloff>:/image/images/insertText.png</iconset>
   </property>
   <property name="text">
    <string>insertText</string>
   </property>
  </action>
  <action name="actionDefault_save_location">
   <property name="text">
    <string>Default save location</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../writernote.qrc"/>
 </resources>
 <connections/>
</ui>
